src/components/sidebar/Sidebar.scss

```diff
@@ -12,7 +12,7 @@
     }
 
     .hireflow-icon {
-        width: 100%;
+        width: 75%;
```

Comment:
I don't know if this had the intended effects I expected, can you change this to a set 200 px?